provider/core/src/marker.rs

```diff
@@ -440,6 +440,9 @@ pub struct DataMarkerInfo {
     /// attributes during provider export.
     #[cfg(feature = "export")]
     pub attributes_domain: &'static str,
+    /// Whether to create constants for each data struct in baked data.
+    #[cfg(feature = "export")]
```

Comment:
thought (non blocking): not actually convinced we should be `cfg()`ing this info. I guess the worry is they show up in binaries? Would be ideal if this metadata was always obtainable from the marker type without a feature. But not a problem for this PR.

Reply:
That's valid; I'd love for a design where the thing we need to keep small is a subset of DMI so that metadata isn't configured. But this is a very minor wish.